iam-login-service/src/main/java/it/infn/mw/iam/api/scim/provisioning/ScimUserProvisioning.java

```diff
@@ -112,59 +112,56 @@ public ScimUserProvisioning(IamAccountService accountService,
 
 
   // All the supported attribute Operators
-  private enum AttributeOperators{
-    eq,
-    co
+  private enum AttributeOperators {
+    eq, co
   }
 
   // All the supported filter attributes
-  private enum Attributes{
+  private enum Attributes {
     // All related to the name of the user
-    familyName, 
-    givenName,
-    username,
+    familyName, givenName, username,
 
     // Other info
-    active, 
-    emails
+    active, emails
   }
 
 
 
-
   private ArrayList<String> parseFilters(final String filtersParameter) {
 
     ArrayList<String> result = new ArrayList<>();
 
     if (!Strings.isNullOrEmpty(filtersParameter)) {
-      // Iterating over all defined enums 
-      for(AttributeOperators operator : AttributeOperators.values()){
+      // Iterating over all defined enums
+      for (AttributeOperators operator : AttributeOperators.values()) {
 
         int from = 0;
 
         // Need to look at all instances that corresponds to the operator
-        while(-1 != filtersParameter.indexOf(operator.toString(), from)){
+        while (-1 != filtersParameter.indexOf(operator.toString(), from)) {
 
-           // Looking for the first instance of the given operator in the request
+          // Looking for the first instance of the given operator in the request
           int index = filtersParameter.indexOf(operator.toString(), from);
 
-          index = index==0?index=1:index;
+          index = index == 0 ? index = 1 : index;
```

Comment:
this I don't understand. does it mean that if index is 0, it should become 1? maybe an `if` is clearer. and in any case I don't think you need `index = ` before `1`.

Reply:
Yes, if index is equal to 0 than it should become 1. It is to make sure it does not get stuck by finding the next instance and keep moving it forward. 
Yes, one can omit the `index = `  before the `1`, haven't been using shorthand that much :sweat_smile:
I can do an `if` if that is preferred, just did the shorthand as it is shorter, line-wise

Reply:
In this case I think an `if` would be clearer
```suggestion
          if (index == 0) {
            index = 1;
          };
```

Reply:
Sure thing, I commited your suggestion